project/Makefile

```diff
@@ -20,3 +20,7 @@ test:
 
 docs:
 	cd {{ docs_path }} && sphinx-build -W -b html -d _build/doctrees . _build/html
+
+assets:
+	cd Resources/assets_src && npm install
+	cd Resources/assets_src && gulp
```

Comment:
So… `cd Resources/assets_src` twice? This does not look right. Maybe do this instead : 

```bashrc
pushd Resources/assets_src && npm install && popd
pushd Resources/assets_src && gulp && popd
```